### PR TITLE
Add `CODE_OF_CONDUCT.md` and `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by getsource**
First pass.

Adds initial Code of Conduct and Contributing files, along
with minor changes in the readme files for linking and readability.

Code of Conduct based on https://github.com/WordPress/hosting-handbook/blob/master/CODE_OF_CONDUCT.md
Contributing file based on https://github.com/WordPress/wpdev-docker-images/blob/master/CONTRIBUTING.md

See #79.

### CODE REVIEW

**Comment by getsource** [comment]
Looking at the initial ticket, looks like I commented that it'd be good to add a note about the coding standards:
https://github.com/WordPress/phpunit-test-reporter/issues/79#issuecomment-634544732

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+== Contributing Guidelines ==
```
Comment (pfefferle) [inline CONTRIBUTING.md:1]: This is the WordPress `readme.txt` style... should be `# Contributing Guidelines` ;)

Comment (getsource) [reply CONTRIBUTING.md]: Good catch -- thanks! Updated.

**Review by pfefferle — APPROVED** [review]
LGTM 👍

**Pull request merged**